src/types/numeric.md

```diff
@@ -40,6 +40,10 @@ r[type.numeric.int.size.usize]
 The `usize` type is an unsigned integer type with the same number of bits as the
 platform's pointer type. It can represent every memory address in the process.
 
+> [!NOTE]
+> While a `usize` can represent every *address*, converting a *pointer* to a `usize` is not necessarily a reversible operation.
+> For more information, see the documentation for [type cast expressions] and [`std::ptr`].
```

Comment:
```suggestion
> For more information, see the documentation for [type cast expressions], [`std::ptr`], and [provenance](https://doc.rust-lang.org/std/ptr/index.html#provenance) in particular.
```

Reply:
I was steering clear of writing “provenance” in the text, but perhaps that is too cautious. Added.